### PR TITLE
Update effects 

### PR DESCRIPTION
**Pull request by jeremylarner**
Changed effects.js to match L5R attributes, and added some effects we will need for implementing cards.

### CODE REVIEW

**Comment by gryffon** [comment]
I've re-run the pull request build, and it's now actual failures.

**Comment by jeremylarner** [comment]
This passes now

File: server/game/drawcard.js [hunk 1]
```diff
@@ -106,6 +105,20 @@ class DrawCard extends BaseCard {
         }
     }
 
+    modifyGlory(amount, applying = true) {
+        /**
+         * Modify the military skill.
```
Comment (gryffon) [inline server/game/drawcard.js]: Comment looks it needs to be updated.

**Comment by jeremylarner** [comment]
Thanks for spotting that.  I think it's good to go now.

**Pull request merged**